Feature/mast 5 user settings

TomasDrozdik opened the pull request:
There is some work necessary regarding spacing in the main page, however I would appreciate your help with that because I wasn't able to successfully deal with it.

TomasDrozdik committed:
Extract form fields from login css into separate file

TomasDrozdik committed:
Update form helpers macros

TomasDrozdik committed:
Add user settings forms

TomasDrozdik committed:
Add brief form validation

TomasDrozdik committed:
Put some spacing intfo radio form field

TomasDrozdik committed:
Fix picture file name

JanKleprlik commented on mast/models.py:
```diff
@@ -1 +1,34 @@
+        self.verified = False
+        self.email = email
+
+    def verify_profile(self, first_name, last_name, age, sex, employee, nickname=None):
```
Přejmenoval bych metodu na create_profile, soudě podle toho co dělá

TomasDrozdik replied:
Jo toto sú len mockupy na overenie toho čo robí user settings. Takže toto sa celé vymaže keď príde current_user zo session ₊ databáza.

Commit:
Misc fix

TomasDrozdik committed:
Updated user settings forms

MichalKyjovsky commented on mast/templates/personal_dashboard.html:
```diff
@@ -1,21 +1,23 @@
 {% extends "main_layout_template.html" %}
```
Please use modification from my most recent branch in order to avoid conflicts.

TomasDrozdik replied:
Not sure what modification are you referring to since I did not modify this file it should be automatically  updated by your branch commit.

MichalKyjovsky commented on mast/templates/register.html:
```diff
@@ -14,13 +14,11 @@ <h1 class="h2 mb-3">
     </div>
 
-    {{ render_field(form.username, autofocus=true) }}
+    {{ render_text_field(form.email, autofocus=true) }}
```
Please use your defined constraints in form.py also on the frontend. We want to provide the user pre-evaluation before he sends the request on the server. (Obviously, password equality must be validated via Javascript, but this might be omitted.)

MichalKyjovsky commented on mast/templates/user_settings.html:
```diff
@@ -1,3 +1,76 @@
 {% extends "main_layout_template.html" %}
```
Same as above. Please provide some kind of validation defined in forms.py on the frontend. Fill free to refer to: https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation

TomasDrozdik committed:
Add form validation.

- validate emails i.e. input fields with id 'email' and type='email'
- validate matching passwords i.e. input fields with id 'password' and 'confirm_password'
